include/mitsuba/core/argparser.h

```diff
@@ -89,6 +89,12 @@ class MI_EXPORT_LIB ArgParser {
     };
 
 public:
+
+    ~ArgParser() {
+        for (Arg *arg : m_args)
+            delete arg;
```

Comment:
This assumes that `ArgParser::add()` takes ownership of the created `Arg`, which I think is fair.

Reply:
Yep, that's how it is intended.